includes/entities/class-fs-plugin.php

```diff
@@ -59,6 +59,14 @@ class FS_Plugin extends FS_Scope_Entity {
 		 * @var bool
 		 */
 		public $is_live;
+		/**
+         * @author Leo Fajardo (@leorw)
+         *
+		 * @since 1.2.1.7.2
+		 *
+		 * @var string
+		 */
+		public $affiliation;
```

Comment:
@fajardoleo from what I've seen, this value is set by the init SDK params, and the 'affiliation' value is actually boolean. If that's the case, I suggest to rename it to `$has_affiliate_program` (also respectively modify the init_sdk param to `has_affiliate_program` instead of `has_affiliation` and the getter method in this class).